public/stylesheets/navbar.css

```diff
@@ -0,0 +1,108 @@
+/* General Navbar Styles */
+.navbar {
+    background-color: black; /* Black background for the navbar */
+    transition: background-color 0.3s ease-in-out;
+    height: 60px; /* Set consistent height */
+    width: 100%; /* Ensure navbar takes full width */
+}
+
+.navbar-brand {
+    font-weight: bold; /* Bold CampX logo */
+    color: white; /* White logo text */
+    font-size: 2rem;
+}
+
+.navbar-brand:hover {
+    color: white; /* Ensure the CampX logo stays white on hover */
+}
+
+.navbar-brand #rotateX {
+    display: inline-block;
+    animation: rotation 3s infinite;
+    color: white; /* Color of rotating X */
+}
+
+.navbar-nav .nav-link {
+    color: rgba(255, 255, 255, 0.5); /* Semi-transparent text */
+    font-weight: 700;
+    margin-left: 1rem;
+    transition: color 0.3s ease;
```

Comment:
what is the use of this transition?

Reply:
The transition: color 0.3s ease; was applied to ensure that any change in the text color (such as on hover or interaction) would happen smoothly over 0.3 seconds, rather than instantly. This subtle animation improves the visual feedback for users, making the interface feel more responsive and polished. However, since it's removed now, the color will change instantly without the smooth transition.